functions/awsUploader/index.js

```diff
@@ -10,34 +10,14 @@ exports.createHandler = async (snap, bucket, aws, context) => {
   const metadata = snap.data();
   const durationSeconds = Math.ceil(metadata.duration);
```

Comment:
This doesn't look like it's being used in this file anymore - which, if that's ok, means I'd probably not define here.